vendor/perfect-scrollbar/main.scss

```diff
@@ -0,0 +1,123 @@
+// Colors
+$ps-rail-hover: #eee;
+$ps-bar-default: #aaa;
+$ps-bar-hover: #999;
+
+// Helper mixins
+@mixin border-radius($r) {
+  -webkit-border-radius: $r;
+     -moz-border-radius: $r;
+      -ms-border-radius: $r;
+          border-radius: $r;
+}
+
+@mixin transition($t...) {
+  -webkit-transition: $t;
+     -moz-transition: $t;
+       -o-transition: $t;
+          transition: $t;
+}
+
+// Scrollbar mixins
+@mixin scrollbar-rail-default {
+  display: none;
+  position: absolute; /* please don't change 'position' */
+  @include border-radius(4px);
+  opacity: 0;
+  @include transition(background-color 0.2s linear, opacity 0.2s linear);
+}
+
+@mixin scrollbar-rail-hover {
+  background-color: $ps-rail-hover;
+  opacity: 0.9;
+}
+
+@mixin scrollbar-default {
+  position: absolute; /* please don't change 'position' */
+  background-color: $ps-bar-default;
+  @include border-radius(4px);
+  @include transition(background-color 0.2s linear);
+}
+
+@mixin scrollbar-hover {
+  background-color: $ps-bar-hover;
+}
+
+@mixin in-scrolling {
+  &.ps-in-scrolling {
+    pointer-events: none;
+    &.ps-x>.ps-scrollbar-x-rail{
+      @include scrollbar-rail-hover;
+      >.ps-scrollbar-x {
+        @include scrollbar-hover;
+      }
+    }
+    &.ps-y>.ps-scrollbar-y-rail {
+      @include scrollbar-rail-hover;
+      >.ps-scrollbar-y {
+        @include scrollbar-hover;
+      }
+    }
+  }
+}
+
+.ps-container {
+  -ms-touch-action: none;
+  overflow: hidden !important;
+
+  &.ps-active-x > .ps-scrollbar-x-rail,
+  &.ps-active-y > .ps-scrollbar-y-rail {
+    display: block;
+  }
+
+  @include in-scrolling;
```

Comment:
Expected item on line 73 to appear before line 65. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets